code/datums/gamemode/role/catbeast.dm

```diff
@@ -13,6 +13,9 @@
 /datum/role/catbeast/Greet()
 	to_chat(antag.current, "<B><span class='warning'>You are a mangy catbeast!</span></B>")
 	to_chat(antag.current, "The longer you avoid the crew, the greater danger the station will attract! You will generate threat for each new room you enter and for being alive (up to 5 minutes).")
+	to_chat(antag.current, "<span class='warning'>You are also carrying some nasty diseases. Check your notes to see their specs.</span>")
+	to_chat(antag.current, "<span class='notice'>You can accelerate their progression by drinking some milk along with some water, so go find some.</span>")
```

Comment:
This is a little nitpicky but to reduce screen spam I would try to slim this down as much as possible. The more text you give someone when they spawn (the moment they are most panicked) the less they will  read. I'd save the milk/water thing for the wiki and just go with:
**The diseases you are carrying were added to your notes.**